Std/Data/Sum/Lemmas.lean

```diff
@@ -0,0 +1,254 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Yury G. Kudryashov
+-/
+
+import Std.Data.Sum.Basic
+import Std.Tactic.Ext
+
+/-!
+# Disjoint union of types
+
+Theorems about the definitions introduced in `Std.Data.Sum.Basic`.
+-/
+
+namespace Sum
+
+@[simp] protected theorem «forall» {p : α ⊕ β → Prop} :
+    (∀ x, p x) ↔ (∀ a, p (inl a)) ∧ ∀ b, p (inr b) :=
+  ⟨fun h => ⟨fun _ => h _, fun _ => h _⟩, fun ⟨h₁, h₂⟩ => Sum.rec h₁ h₂⟩
+
+@[simp] protected theorem «exists» {p : α ⊕ β → Prop} :
+    (∃ x, p x) ↔ (∃ a, p (inl a)) ∨ ∃ b, p (inr b) :=
+  ⟨ fun
+    | ⟨inl a, h⟩ => Or.inl ⟨a, h⟩
+    | ⟨inr b, h⟩ => Or.inr ⟨b, h⟩,
+    fun
+    | Or.inl ⟨a, h⟩ => ⟨inl a, h⟩
+    | Or.inr ⟨b, h⟩ => ⟨inr b, h⟩⟩
+
+theorem forall_sum_pi {γ : α ⊕ β → Sort _} (p : (∀ ab, γ ab) → Prop) :
```

Comment:
```suggestion
theorem forall_sum {γ : α ⊕ β → Sort _} (p : (∀ ab, γ ab) → Prop) :
```
I'm not sure what the `pi` is referring to here, and in any case we don't use `Pi` to refer to the pi-type anymore (at least in core, `PiInstances.lean` may beg to differ).